api/v0api/full.go

```diff
@@ -161,6 +161,10 @@ type FullNode interface {
 	// If oldmsgskip is set, messages from before the requested roots are also not included.
 	ChainExport(ctx context.Context, nroots abi.ChainEpoch, oldmsgskip bool, tsk types.TipSetKey) (<-chan []byte, error) //perm:read
 
+	ChainExportRange(ctx context.Context, head, tail types.TipSetKey, cfg api.ChainExportConfig) (<-chan []byte, error) //perm:read
+
+	ChainExportRangeInternal(ctx context.Context, head, tail types.TipSetKey, cfg api.ChainExportConfig) error //perm:read
+
```

Comment:
Do we really need in on v0 api?

Reply:
I don't know, is that like the older version kept for compatibility? Then I'd say no...

Reply:
https://github.com/filecoin-project/lotus/pull/10145/commits/71b478e38fb835ef4572b0157feb0e8b4927559d